python/tank/deploy/tank_commands/install.py

```diff
@@ -430,9 +433,9 @@ def run_interactive(self, log, args):
             log.info("Comment and structure preserving mode")
             log.info("-------------------------------------")
             log.info("")
-            log.info("If you add a --preserve-yaml flag, existing comments and "
-                     "structure will be preserved as the yaml files are updated. "
-                     "This is an experimental setting and therefore disabled by default.")            
+            log.info("If you add a --use-pyyaml flag, the original, non-structure-preserving "
```

Comment:
One final comment about the flag. Legacy might not be a super obvious term, but it better reflects the fact that it was previous behaviour and that now it has changed. use-pyyaml feels too "implementation detaily".